### PR TITLE
Force portrait orientation

### PR DESCRIPTION
**Pull request by k88hudson**

### CODE REVIEW

**Comment by thisandagain** [comment]
Need to include `xml2js` in the package.json

File: scripts/after_prepare_android.js [hunk 1]
```diff
@@ -1,8 +1,25 @@
...
+    var Q = context.requireCordovaModule('q');
+    var deferral = new Q.defer();
+
     var indexFile = 'platforms/android/assets/www/index.html';
+    var androidManifestPath = 'platforms/android/AndroidManifest.xml';
     var indexText = fs.readFileSync(indexFile, {encoding: 'utf8'});
     indexText = indexText.replace('<base href="/">', '<base href="/android_asset/www/">');
     fs.writeFileSync(indexFile, indexText);
+    var androidManifest = fs.readFileSync(androidManifestPath, {encoding: 'utf8'});
+    androidManifest = parseString(androidManifest, function (err, result) {
```
Comment (jbuck) [inline scripts/after_prepare_android.js]: You don't need to assign this variable

File: scripts/after_prepare_android.js [hunk 2]
```diff
@@ -1,8 +1,25 @@
...
+        if (err) return deferral.resolve(error);
```
Comment (jbuck) [inline scripts/after_prepare_android.js]: Could you add curly braces?

File: scripts/after_prepare_android.js [hunk 3]
```diff
@@ -1,8 +1,25 @@
...
+        result.manifest.application[0].activity[0].$['android:screenOrientation'] = 'portrait';
```
Comment (jbuck) [inline scripts/after_prepare_android.js]: lol xml

File: scripts/after_prepare_android.js [hunk 4]
```diff
@@ -1,8 +1,25 @@
...
+        var xml = builder.buildObject(result);
```
Comment (jbuck) [inline scripts/after_prepare_android.js]: can you use a different name for this variable, so it doesn't shadow the module?

**Comment by jbuck** [comment]
@k88hudson r+ with nits noted in the PR

File: scripts/after_prepare_android.js (hunk 1 above)
Comment (k88hudson) [inline scripts/after_prepare_android.js]: trueee

File: scripts/after_prepare_android.js (hunk 2 above)
Comment (k88hudson) [inline scripts/after_prepare_android.js]: no hahahahahha.

File: scripts/after_prepare_android.js (hunk 3 above)
Comment (k88hudson) [inline scripts/after_prepare_android.js]: yep.

File: scripts/after_prepare_android.js (hunk 4 above)
Comment (k88hudson) [inline scripts/after_prepare_android.js]: AH good call wow

**Pull request merged**